brutils/phone.py

```diff
@@ -143,31 +176,28 @@ def _is_valid_mobile(phone_number):  # type: (str) -> bool
     )
 
 
-def remove_international_code_phone(phone_number):  # type: (str) -> str
+def _generate_ddd_number():  # type() -> str
     """
-    Function responsible for remove a international code phone
-
-    Args:
-        phone_number (str): The phone number with international code phone.
+    Generate a valid DDD number.
+    """
+    return f'{"".join([str(randint(1, 9)) for i in range(2)])}'
 
-    Returns:
-        str: The phone number without international code
-            or the same phone number.
 
-    >>> remove_international_code_phone("5511994029275")
-    '11994029275'
-    >>> remove_international_code_phone("1635014415")
-    '1635014415'
-    >>> remove_international_code_phone("+5511994029275")
-    '+11994029275'
+def _generate_mobile_phone():
     """
+    Generate a valid and random mobile phone number
+    """
+    ddd = _generate_ddd_number()
+    client_number = [str(randint(0, 9)) for i in range(8)]
 
-    pattern = r"\+?55"
+    phone_number = f'{ddd}9{"".join(client_number)}'
 
-    if (
-        re.search(pattern, phone_number)
-        and len(phone_number.replace(" ", "")) > 11
-    ):
-        return phone_number.replace("55", "", 1)
-    else:
-        return phone_number
+    return phone_number
+
+
+def _generate_landline_phone():  # type () -> str
+    """
+    Generate a valid and random landline phone number.
+    """
+    ddd = _generate_ddd_number()
+    return f"{ddd}{randint(2,5)}{str(randint(0,9999999)).zfill(7)}"
```

Comment:
Só adicionando uma linha vazia no final do arquivo:

```suggestion
    return f"{ddd}{randint(2,5)}{str(randint(0,9999999)).zfill(7)}"
 
```